File: Programmers/Level1/없는 숫자 더하기.py
def solution(numbers):
    all_numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
    no_numbers = []

    for n in all_numbers:
        if n not in numbers:
            no_numbers.append(n)

    answer = sum(no_numbers)
    return answer

File: Swift 코딩테스트 연습/없는 숫자 더하기.swift
import Foundation

func solution(_ numbers:[Int]) -> Int {
    var zeroToTen:Set<Int> = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
    var answer:Int = 0
    
    for n in numbers {
        zeroToTen.remove(n)
    }
    
    zeroToTen.forEach {
        answer += $0
    }
    
    return answer
}